pages/open-source.tsx

```diff
@@ -317,50 +317,52 @@ OSS.pageId = PageId.OSS;
 export default OSS;
 
 export const getStaticProps: GetStaticProps = async () => {
-  const orgName = "thirdweb-dev";
-
+  // Array of accounts to be tracked
+  const accounts = ["thirdweb-dev", "thirdweb-example"];
   const authHeader = {
     headers: {
       Authorization: `token ${process.env.GITHUB_API_TOKEN}`,
     },
   };
 
-  // Fetch the list of all repositories belonging to the organization
-  const reposResponse = await fetch(
-    `https://api.github.com/orgs/${orgName}/repos?per_page=100`,
-    authHeader,
-  );
-
-  const reposData = (await reposResponse.json()) as GithubRepository[];
-
-  const repos = reposData
-    .filter((repo) => repo.fork === false)
-    .filter((repo) => repo.name !== "shopify-thirdweb-theme")
-    .map((repo) => repo.name);
-
   const contributors: Record<string, GithubContributor> = {};
 
-  for (const repo of repos) {
-    const response = await fetch(
-      `https://api.github.com/repos/${orgName}/${repo}/contributors`,
+  for (const account of accounts) {
+    // Fetch the list of all repositories belonging to the organization
+    const reposResponse = await fetch(
+      `https://api.github.com/orgs/${account}/repos?per_page=100`,
       authHeader,
     );
-    const data = (await response.json()) as GithubContributor[];
 
-    data.forEach((contributor) => {
-      const login = contributor.login;
-      const contributions = contributor.contributions;
-      if (contributors[login]) {
-        contributors[login].contributions += contributions;
-      } else {
-        contributors[login] = {
-          login,
-          avatar_url: contributor.avatar_url,
-          html_url: contributor.html_url,
-          contributions,
-        };
-      }
-    });
+    const reposData = (await reposResponse.json()) as GithubRepository[];
+
+    const repos = reposData
+      .filter((repo) => repo.fork === false)
+      .filter((repo) => repo.name !== "shopify-thirdweb-theme")
+      .map((repo) => repo.name);
+
+    for (const repo of repos) {
```

Comment:
same thing here this should be done in parallel instead of one at a time (using promise.all)